pkg/zcache/redis_cache.go

```diff
@@ -85,3 +93,33 @@ func (c *redisCache) HSet(ctx context.Context, key string, values ...interface{}
 func (c *redisCache) HGet(ctx context.Context, key, field string) (string, error) {
 	return c.client.HGet(ctx, key, field).Result()
 }
+
+func (c *redisCache) GetStats() ZCacheStats {
+	poolStats := c.client.PoolStats()
+	// ctx := context.Background()
```

Comment:
Are these comments useful?

Reply:
Not really... we can take statistics from Redis directly with INFO command. After that, we can parse it and take what we want. I left them there in order to complete it later